### PR TITLE
[2.19.x] Fixed Result Forms from hiding temporal attributes

### PR DESCRIPTION
**Pull request by bdeining**
#### What does this PR do?
Fixed Result Forms from hiding temporal attributes.  
#### Who is reviewing it? 

@cassandrabailey293 @andrewzimmer 
#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@bdthomson
#### How should this be tested?

Build / Install / Verify that temporal attributes are present in the result forms dropdown and verify basic search forms still work as expected.
#### What are the relevant tickets?
Fixes: #____

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/catalog-ui-search/src/main/webapp/component/singletons/metacard-definitions.js [hunk 1]
```diff
@@ -84,15 +84,21 @@ const metacardStartingTypes = {
...
-  metacardStartingTypes[proposedType] = {
-    id: proposedType,
-    type: 'DATE',
-    alias: properties.attributeAliases[proposedType],
-    hidden: properties.isHidden(proposedType),
-  }
-})
+function metacardStartingTypesWithTemporal() {
+  // needed to handle erroneous or currently unknown attributes (they could be picked up after searching a source)
+  let metacardStartingTypeWithTemporal = { ...metacardStartingTypes }
```
Comment (willwill96) [inline ui/packages/catalog-ui-search/src/main/webapp/component/singletons/metacard-definitions.js]: :pencil2: const

**Comment by bdeining** [comment]
> Approved, but what was happening before?

The temporal attributes were hidden from the result form.  Essentially need to split the list up for two different use cases.

**Comment by bdeining** [comment]
build now

**Comment by zta6** [comment]
🎉 hero successful 🎉 
verified temporal fields showed up in result forms dropdown
verified basic searches still function as expected

**Pull request merged**